### PR TITLE
PMM-2426 Page not found appears on main page

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 NEW_VERSION_FILE = SCRIPT_DIR + '/VERSION'
 OLD_VERSION_FILE = GRAFANA_DB_DIR + '/PERCONA_DASHBOARDS_VERSION'
 HOST             = 'http://127.0.0.1:3000'
+AUTH_HOST        = 'http://admin:admin@127.0.0.1:3000'
```
Comment (delgod) [inline import-dashboards.py]: admin user can be removed on end-user installation, so we can rely only on api_key

File: import-dashboards.py [hunk 2]
```diff
@@ -254,6 +264,18 @@ def set_home_dashboard(api_key):
     # cur.execute("REPLACE INTO preferences (id, org_id, user_id, version, home_dashboard_id, timezone, theme, created, updated) "
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
+def set_org_timezone(api_key):
+    r = requests.get('%s/api/org/preferences' % (HOST,), headers=grafana_headers(api_key))
+    print ' * Organization preferences: %r %r' % (r.status_code, r.content)
+    if r.status_code != 200:
```
Comment (askomorokhov) [inline import-dashboards.py]: Please use `HTTPStatus.OK` form `http` lib instead of magic number.

Comment (adivinho) [reply import-dashboards.py]: It has been replaced by a constant from httplib.

File: import-dashboards.py (hunk 1 above)
Comment (delgod) [reply import-dashboards.py]: still actual

**Pull request closed without merging**